README.md

```diff
@@ -38,6 +38,7 @@ There are a couple of ways you can contribute to this repo:
 - **Documentation**: Found a typo or strangely worded sentences? Submit a PR!
 - **Code**: Contribute bug fixes, features or design changes:
   - Clone the repository locally and open in VS Code.
+  - Have [TSLint for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=eg2.tslint) extension installed in VS Code.
```

Comment:
I would prefer:
- Install [TSLint for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=eg2.tslint)

(It's simpler and matches all of the other bullets - which start with a verb)

Reply:
sure